### PR TITLE
Add exporter type none for disabling it

### PR DESCRIPTION
**Pull request by shashank11p**

### CODE REVIEW

File: proto/hypertrace/agent/config/v1/config.proto [hunk 1]
```diff
@@ -134,6 +137,9 @@ enum MetricReporterType {
     // Prometheus exposition format.
     // see https://github.com/prometheus/docs/blob/main/content/docs/instrumenting/exposition_formats.md
     METRIC_REPORTER_TYPE_PROMETHEUS = 2;
+
+    // Disable metric reporting
+    METRIC_REPORTER_TYPE_NONE = 3;
```
Comment (davexroth) [inline proto/hypertrace/agent/config/v1/config.proto:142]: while we're here, should we also add a logging exporter?

Comment (jcchavezs) [reply proto/hypertrace/agent/config/v1/config.proto:142]: Curious about logger exporting for metrics. Is that a thing?

Comment (davexroth) [reply proto/hypertrace/agent/config/v1/config.proto:142]: yep, at least for the collector.

https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/loggingexporter/README.md

Comment (ryandens) [reply proto/hypertrace/agent/config/v1/config.proto:142]: Also for java: https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md#logging-exporter

Comment (shashank11p) [reply proto/hypertrace/agent/config/v1/config.proto]: yes, done.

File: ENV_VARS.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ Agents can be configured using environment variables:
 | HT_REPORTING_SECURE | When `true`, connects to endpoints over TLS. |
 | HT_REPORTING_TOKEN | User specific token to access Traceable API |
 | HT_REPORTING_CERT_FILE | Certificate file containing the CA to verify the server's certificate. This is for private certificates. If this is set then `secure` above should also be set to `true`. |
```
Comment (ryandens) [inline ENV_VARS.md:14]: Should we add env var entries in this file for `HT_REPORTING_TRACE_REPORTER_TYPE` and `HT_REPORTING_METRIC_REPORTER_TYPE`?

**Review by ryandens — APPROVED** [review]

File: ENV_VARS.md (hunk 2 above)
Comment (shashank11p) [reply ENV_VARS.md:14]: We will need to fix the env var generation script. I dont know why these did not get generated. We should not modify this file manually.

Comment (ryandens) [reply ENV_VARS.md:14]: ah, gotcha. In that case, this is out of scope

**Review by davexroth — APPROVED** [review]

**Pull request merged**